spk/watcher3/Makefile

```diff
@@ -0,0 +1,46 @@
+GIT_REPO = https://github.com/nosmokingbandit/Watcher3.git
+
+SPK_NAME = watcher3
+SPK_VERS = $(shell git ls-remote ${GIT_REPO} HEAD | head -c 8)
```

Comment:
`SPK_VERS` for those kind of packages normaly looks like this `SPK_VERS = $(shell date +%Y%m%d)` since they most likely contain an in app update mechanism...

Reply:
Yes I know that is how it is usually done. But it makes more sense to show wich version (on git) that is used.

Reply:
afaik we use the date approach so that it is clear that the version is not in any kind assosicated with the upstream version... using git version implies that you'll have to update the SPK to get a newer version and we'll get users asking for frequent updates again. that was the main reason why we moved to the date style format. To make clear that that's the version/date of the installer not the actual app...

Reply:
in addition to that: it can get kind of confusing when you install the SPK and then update the app from inside the app itself (over git i guess) so you'll have another version then the one specified in the SPK ;)